docs/toc.md

```diff
@@ -1,164 +1,93 @@
-# [Getting started](getting-started/index.md)
-## [What is the .NET Core Platform?](getting-started/what-is-dotnet.md)
-## [Installing .NET Core](getting-started/installing/index.md)
-### [.NET Core System Requirements](getting-started/installing/system-reqs.md)
-### [Installing .NET Core on Windows](getting-started/installing/installing-core-windows.md)
-### [Installing .NET Core on Linux](getting-started/installing/installing-core-linux.md)
-### [Installing .NET Core on OS X](getting-started/installing/installing-core-osx.md)
-## [Using the CLI tools to write console apps](getting-started/cli-console-app-tutorial.md)
-## [Setting up your integrated development environment](getting-started/devenv/index.md)
-### [🔧 Development using Visual Studio 2015](getting-started/devenv/using-visual-studio.md)
-### [🔧 Development using Visual Studio Code](getting-started/devenv/using-visual-studio-code.md)
-### [🔧 Development using vim](getting-started/devenv/using-vim.md)
-### [🔧 Development using Sublime Text](getting-started/devenv/using-sublime.md)
+# [About .NET](about/index.md)
+## [.NET Products](about/products.md)
+## [Community](about/community.md)
+## [Open Source](about/open-source.md)
+## [Learning Resources](about/learn.md)
+## [Talks](about/talks.md)
+# [Handbook](handbook.md)
+## [.NET Standard](standard-library.md)
+## [Target Frameworks](target-frameworks.md)
+## [Numerics](numerics.md)
+## [Collections and Data Structures](collections/collections-and-data-structures.md)
+### [Selecting a Collection Class](collections/selecting-a-collection-class.md)
+### [Commonly Used Collection Types](collections/commonly-used-collection-types.md)
+### [When to Use Generic Collections](collections/when-to-use-generic-collections.md)
+### [Comparisons and Sorts Within Collections](collections/comparisons-and-sorts-within-collections.md)
+### [Sorted Collection Types](collections/sorted-collection-types.md)
+### [Hashtable and Dictionary Collection Types](collections/hashtable-and-dictionary-collection-types.md)
+### [Thread-Safe Collections](collections/thread-safe-collections.md)
+#### [BlockingCollection Overview](collections/threadsafe/blockingcollection-overview.md)
+#### [When to Use a Thread-Safe Collection](collections/threadsafe/when-to-use-a-thread-safe-collection.md)
+# [Fundamentals](fundamentals/index.md)
```

Comment:
Some of the 2nd level headings here seem very specific ("Create an Object Pool by Using a Concurrent Bag" for example.

Do we want to roll some of these into 'scenarios', or roll the content into slightly higher-level topics. (e.g. Collections, LINQ).